### ISSUE TITLE
Problem List Does Not Clear When Issues are Fixed

### ISSUE THREAD

**Issue by jtsom** [OmniSharp/omnisharp-vscode#5362]
## Issue Description ##
When errors are found and fixed, the problem list is not updated to clear out the issues. Especially if the file that the issue was in has changed names.

Only way to clear problem list is to use Reload Window command, or exit and relaunch VS Code.

## Steps to Reproduce ##
Create project, add new file that references some .NET assembly, but the Using has not been added.
Rename the file, and update add the Using.

## Expected Behavior ##
Errors are removed from Problem list.

## Actual Behavior ##
Errors persist in problem list.

### OmniSharp log ###
<details>Post the output from Output-->OmniSharp log here</details>

### C# log ###
<details>Post the output from Output-->C# here</details>

## Environment information ##

**VSCode version**: 1.72.0-insider
**C# Extension**: 1.25.0

<details><summary>Dotnet Information</summary>
.NET SDK (reflecting any global.json):
 Version:   6.0.400
 Commit:    7771abd614

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22000
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.400\

global.json file:
  Not found

Host:
  Version:      6.0.8
  Architecture: x64
  Commit:       55fb7ef977

.NET SDKs installed:
  3.1.422 [C:\Program Files\dotnet\sdk]
  5.0.408 [C:\Program Files\dotnet\sdk]
  6.0.108 [C:\Program Files\dotnet\sdk]
  6.0.303 [C:\Program Files\dotnet\sdk]
  6.0.400 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.28 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.28 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.7 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Download .NET:
  https://aka.ms/dotnet-download

Learn about .NET Runtimes and SDKs:
  https://aka.ms/dotnet/runtimes-sdk-info
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|azure-account|ms-vscode|0.11.1|
|csharp|ms-dotnettools|1.25.0|
|csharpextensions|kreativ-software|1.7.1|
|gitlens-insiders|eamodio|2022.8.3015|
|hexeditor|ms-vscode|1.9.8|
|hl7tools|RobHolme|1.14.2|
|js-debug-nightly|ms-vscode|2022.8.3017|
|jwt-debugger|yokawasa|0.4.2|
|material-icon-theme|PKief|4.20.0|
|msbuild-project-tools|tintoy|0.4.9|
|ng-template|Angular|14.1.0|
|night-owl|sdras|2.0.1|
|quicktype|quicktype|12.0.46|
|sync-scroll|dqisme|1.3.0|
|TabOut|albert|0.2.1|
|vs-code-xml-format|fabianlauer|0.1.5|
|vscode-azureappservice|ms-azuretools|0.24.3|
|vscode-azurefunctions|ms-azuretools|1.7.4|
|vscode-azureresourcegroups|ms-azuretools|0.5.5|
|vscode-azurestorage|ms-azuretools|0.14.2|
|vscode-base64|adamhartford|0.1.0|
|vscode-cosmosdb|ms-azuretools|0.19.2|
|vscode-thunder-client|rangav|1.18.7|
|vscodeintellicode|VisualStudioExptTeam|1.2.24|;

</details>

**Comment by heartacker** [comment]
me too，

**Comment by m4ss1m0g** [comment]
Same problem; I restart omnisharp -> F1 -> **OmniSharp: Restart Omnisharp**

**Comment by jtsom** [comment]
Agreed. Old problem, still happens. OmniSharp lags way behind changes in the code.
